### DISCUSSION TITLE
new plugin : `tauri-plugin-sharetarget` for Android

### DISCUSSION THREAD

**Discussion by lafleurdeboum** [tauri-apps/tauri#11572] [Show and tell]
I just published a usable version of [`tauri-plugin-sharetarget`](https://gitlab.com/lafleurdeboum/tauri-plugin-sharetarget), a plugin to let developers receive share intents on Android with Tauri. I took quite some time to polish its README, I hope it makes adoption straightforward. The typescript companion package is [`tauri-plugin-sharetarget-api`](https://www.npmjs.com/package/tauri-plugin-sharetarget-api).

Comments welcome !

**Comment by FabianLars** [comment]
awesome, would you mind adding it to https://github.com/tauri-apps/awesome-tauri/ ?
